frontend/app/abtests/BundleVariant.scala

```diff
@@ -49,8 +48,8 @@ object BundleTier {
 
 
 object Distribution {
```

Comment:
Conceptually, using the priceindex to differentiate whether a feature is visible or not is super confusing, especially as the prices are the same on both variants. I know what you're saying, and could call priceindex something else, but that means more changes than are necessary to achieve the results we're after.